upi/aws/cloudformation/01_vpc.yaml

```diff
@@ -244,6 +244,29 @@ Resources:
       DestinationCidrBlock: 0.0.0.0/0
       NatGatewayId:
         Ref: NAT3
+  LoadBalancingEndpoint:
```

Comment:
Again this UPI templates are reference only. And I don't think we allow special env setup to handle these and not add these for all.

This is not required for openshift clusters and don't provide value.